src/screens/AddAccounts/01-SelectCrypto.js

```diff
@@ -50,10 +53,20 @@ class AddAccountsSelectCrypto extends Component<Props, State> {
     ),
   };
 
+  state = {
+    sortedCryptoCurrencies: []
+  }
+
   cryptocurrencies = listCryptoCurrencies(this.props.devMode).concat(
     listTokens(),
   );
 
+  async componentDidMount() {
+    const sortedCryptoCurrencies = await currenciesByMarketcap(this.cryptocurrencies)
+    this.setState({ sortedCryptoCurrencies })
```

Comment:
I kinda would prefer we use this hook: https://github.com/LedgerHQ/ledger-live-common/blob/master/src/currencies/sortByMarketcap.js#L52

because it already starts with the sorted list so it prevent a double rendering reordering flash.